tox.ini

```diff
@@ -62,7 +62,7 @@ commands =
 basepython = python3.8
 
 [testenv:lint]
-basepython = python3.8
+basepython = python3
```

Comment:
You will never install Py38, will you? :smile:

Reply:
I use bb5 python in latest modules